Add a blog to Zazu's documentation site. [skip ci]

bayleedev opened the pull request:
This allows us to talk about videos, walkthroughs, releases and new plugins.

This PR includes:

* A blog link at the top
* Pagination on the blog
* Correct breadcrumbs
* A post about the introduction video
   * This includes the video
   * And a blog-ified version of the content
* Showing the blog in search results

@twang2218, do you have time to review this?

amjith commented on docs/_includes/navigation.html:
```diff
@@ -5,6 +5,8 @@
     <a class="c-navigation__item {% if page.url == '/documentation/' %}is-active{% endif %}" href="/documentation/">Docs</a>
     <a class="c-navigation__item {% if page.url == '/plugins/' %}is-active{% endif %}" href="/plugins/">Plugins</a>
     <a class="c-navigation__item {% if page.url == '/themes/' %}is-active{% endif %}" href="/themes/">Themes</a>
+    <a class="c-navigation__item {% if page.url == '/themes/' %}is-active{% endif %}" href="/themes/">Themes</a>
```
Why do you have 'themes' listed twice?

bayleedev replied:
I really like themes, geez.

amjith commented on docs/_posts/2017-03-07-video-introduction-to-zazu.md:
```diff
@@ -0,0 +1,77 @@
+
+Once it is successfully installed, you can launch Zazu. You’ll notice the
+feather icon in the menu bar of your operating system. You can now open Zazu
+sing the shortcut key `Alt-Space`. Launching an application is as simple as
```
~sing~ using

amjith commented on docs/_posts/2017-03-07-video-introduction-to-zazu.md:
```diff
@@ -0,0 +1,77 @@
+
+## Default Plugins
+
+As I mentioned earlier, Zazu ships with a rich set of plugins that go beyond the
```
Lose `As I mentioned earlier`.

amjith commented on docs/_posts/2017-03-07-video-introduction-to-zazu.md:
```diff
@@ -0,0 +1,77 @@
+
+But the real power of Zazu lies with the plugins contributed by users to satisfy
+their specific needs. You can browse through a list of available plugins on the
+[plugins page][plugins page]or you can look at them straight from Zazu by typing
```
Add space before `or you....`

Commit:
Add a blog to Zazu's documentation site. [skip ci]

This allows us to talk about videos, walkthroughs, releases and new plugins.

This PR includes:

* A blog link at the top
* Pagination on the blog
* Correct breadcrumbs
* A post about the introduction video
   * This includes the video
   * And a blog-ified version of the content
* Showing the blog in search results
